modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/GridDhtTxFinishResponse.java

```diff
@@ -79,41 +79,53 @@ public int miniId() {
         return miniId;
     }
 
+    /** Sets mini future ID. */
+    public void miniId(int miniId) {
+        this.miniId = miniId;
+    }
+
     /**
      * @return Error for check committed backup requests.
      */
-    public Throwable checkCommittedError() {
-        return checkCommittedErr;
+    @Nullable public Throwable checkCommittedError() {
+        ErrorMessage checkCommittedErrMsg = this.checkCommittedErrMsg;
+
+        return checkCommittedErrMsg == null ? null : checkCommittedErrMsg.toThrowable();
     }
 
     /**
      * @param checkCommittedErr Error for check committed backup requests.
      */
-    public void checkCommittedError(Throwable checkCommittedErr) {
-        this.checkCommittedErr = checkCommittedErr;
+    public void checkCommittedError(@Nullable Throwable checkCommittedErr) {
```

Comment:
Actually, there is only one usage,and there error can not be null. 
And alse we can remove this method.

Reply:
Why removing this method?